pkg/dao/rpm.go

```diff
@@ -198,6 +202,20 @@ func (r rpmDaoImpl) Search(orgID string, request api.SearchRpmRequest, limit int
 	return dataResponse, nil
 }
 
+func (r rpmDaoImpl) getGroupUrlOrUuid(urls []string, uuids []string) *gorm.DB {
+	if len(urls) > 0 && len(uuids) > 0 {
+		return r.db.Where("repositories.url in ?", urls).
+			Or("repository_configurations.uuid in ?", uuids)
```

Comment:
When urls/uuids is empty, it seems the query here (for url) would be `"WHERE repositories.url in (NULL)"`, which doesn't find anything but doesn't error. What's the reason for avoiding that with the conditionals here? Is it a bad practice or something?

Reply:
The reason is I didn't try what happens when urls or uuids is empty! :+1: thanks I will review!

Reply:
I have cleaned-up the code and I realized it was more simple; thanks @rverdile